general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -111,11 +117,18 @@ def __init__(
         }
         self.session = requests.Session()
 
+        if cq_token:
+            kwargs["cq_token"] = cq_token
+
+        if use_stored_ibmq_credentials:
+            config = gss.superstaq_client.read_ibm_credentials(ibmq_name or "default-ibm-quantum")
```

Comment:
we should not assume a name here - instead if `read_ibm_credentials()` should accept None (see prior comment)
```suggestion
            config = read_ibm_credentials(ibmq_name)
```